api/api.go

```diff
@@ -712,6 +847,27 @@ func findNetwork(c libnetwork.NetworkController, s string, by int) (libnetwork.N
 	return nw, &successResponse
 }
 
+func findSandbox(c libnetwork.NetworkController, s string, by int) (libnetwork.Sandbox, *responseStatus) {
+	var (
+		sb  libnetwork.Sandbox
+		err error
+	)
+
+	switch by {
+	case byID:
+		sb, err = c.SandboxByID(s)
+	default:
```

Comment:
It's just to maintain the same construct I have for network and endpoint.
So that all the GET serving function look alike and follow the same steps when looking for the resource.